crypto/src/test/java/org/hyperledger/besu/crypto/SECP256R1Test.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.crypto;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hyperledger.besu.crypto.Hash.keccak256;
+
+import java.io.File;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SECP256R1Test {
+
+  protected SECP256R1 secp256R1;
+
+  protected static String suiteStartTime = null;
+  protected static String suiteName = null;
+
+  @BeforeClass
+  public static void setTestSuiteStartTime() {
+    suiteStartTime =
+        LocalDateTime.now(ZoneId.systemDefault())
+            .format(DateTimeFormatter.ofPattern("yyyyMMdd-HHmmss"));
+    suiteName(SECP256K1Test.class);
+    ;
```

Comment:
Don't need this extra semicolon.

Reply:
(please remove it from `SECP256K1Test` as well) :)

Reply:
:+1: